src/state/reducers/serviceTypeReducer.js

```diff
@@ -0,0 +1,149 @@
+import {
+  GET_ALL_SERVICE_TYPE_START,
+  GET_ALL_SERVICE_TYPE_SUCCESS,
+  GET_ALL_SERVICE_TYPE_FAIL,
+  GET_ALL_SERVICE_TYPE_RESOLVE,
+  GET_SERVICE_TYPE_START,
+  GET_SERVICE_TYPE_SUCCESS,
+  GET_SERVICE_TYPE_FAIL,
+  GET_SERVICE_TYPE_RESOLVE,
+  ADD_SERVICE_TYPE_START,
+  ADD_SERVICE_TYPE_SUCCESS,
+  ADD_SERVICE_TYPE_FAIL,
+  ADD_SERVICE_TYPE_RESOLVE,
+  EDIT_SERVICE_TYPE_START,
+  EDIT_SERVICE_TYPE_SUCCESS,
+  EDIT_SERVICE_TYPE_FAIL,
+  EDIT_SERVICE_TYPE_RESOLVE,
+  DELETE_SERVICE_TYPE_START,
+  DELETE_SERVICE_TYPE_SUCCESS,
+  DELETE_SERVICE_TYPE_FAIL,
+  DELETE_SERVICE_TYPE_RESOLVE,
+} from '../actions/ServiceTypeActions';
+
+export const initialServiceTypeState = {
+  types: [],
+  type: null,
```

Comment:
Please name these. seriveTypes:[], and serviceType:null. Naming of 'type' is not descriptive enough